autopush/router/webpush.py

```diff
@@ -20,6 +20,8 @@
 from autopush.router.interface import RouterException, RouterResponse
 from autopush.router.simple import SimpleRouter
 
+TTL_URL = "https://webpush-wg.github.io/webpush-protocol/#rfc.section.6.2"
```

Comment:
I was considering that, I admit I was hoping this would be short-lived enough that it'll be removed entirely before we consider adding more (later we'll link to our docs, which will have rich information on all the error codes).